guardrails/logger.py

```diff
@@ -21,6 +21,8 @@ def __init__(self, level=logging.NOTSET, scope=base_scope):
 
     def emit(self, record: LogRecord) -> None:
         logs = self.scoped_logs.get(self.scope, [])
+        print("record in logs!!!!", record)
```

Comment:
This does not ever run, from the tests I've run.

The print statements in guard.py do print, so I know for a fact that logger.debug is running.

Reply:
Talked about this offline, had to do with the log level value